samples/sampler/stories/default/ColorPicker.js

```diff
@@ -17,8 +17,8 @@ storiesOf('Agate', module)
 		() => (
 			<ColorPicker
 				direction={select('direction', prop.direction, ColorPicker, 'right')}
-				defaultValue={text('defaultValue', ColorPicker, '#3467af')}
 				onChange={action('onChange')}
+				defaultValue={text('value', ColorPicker, '#3467af')}
```

Comment:
Suggest we remove this knob altogether. Setting the `defaultValue` doesn't add much value for exploring how the component looks and behaves which is the purpose of the sample. We can document that the `defaultValue` may be set in the inline docs.